.gitlab-ci.yml

```
@@ -366,100 +366,118 @@ trigger_devops:
-      ]
+      export PIVCAC_CONFIG=$(cat <<EOF
+      {
+        "kubernetesReviewApp": "true",
+        "clientCertS3Bucket": "login-gov-pivcac-public-cert-reviewapp.894947205914-us-west-2",
+        "postgres": {
+          "sslmode": "prefer",
+          "name": "idp",
+          "host": "$CI_ENVIRONMENT_SLUG-login-chart-pivcac-pg.review-apps",
+          "username": "postgres",
+          "password": "pivcac"
+        },
+        "idpHost": "$CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov",
+        "domainName": "$CI_ENVIRONMENT_SLUG-review-app.pivcac.identitysandbox.gov"
+      }
       EOF
       )
     - |-
-      export DASHBOARD_ENV=$(cat <<EOF
-      [
-        {"name": "KUBERNETES_REVIEW_APP", "value": "true"},
-        {"name": "POSTGRES_SSLMODE", "value": "prefer"},
-        {"name": "POSTGRES_DB", "value": "dashboard"},
-        {"name": "POSTGRES_HOST","value": "$CI_ENVIRONMENT_SLUG-login-chart-dashboard-pg.review-apps"},
-        {"name": "POSTGRES_USERNAME", "value": "postgres"},
-        {"name": "POSTGRES_PASSWORD", "value": "postgres"},
-        {"name": "NEW_RELIC_ENABLED", "value": "false"},
-        {"name": "SAML_SP_ISSUER", "value": "https://$CI_ENVIRONMENT_SLUG-review-app-dashboard.review-app.identitysandbox.gov"},
-        {"name": "IDP_URL", "value": "https://$CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov"},
-        {"name": "IDP_SP_URL", "value": "https://$CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov"},
-        {"name": "POST_LOGOUT_URL", "value": "https://$CI_ENVIRONMENT_SLUG-review-app-dashboard.review-app.identitysandbox.gov"},
-        {"name": "DOMAIN_NAME", "value": "$CI_ENVIRONMENT_SLUG-review-app-dashboard.review-app.identitysandbox.gov"}
-      ]
+      export DASHBOARD_CONFIG=$(cat <<EOF
+      {
+        "kubernetesReviewApp": "true",
+        "postgres": {
+          "sslmode": "prefer",
+          "name": "dashboard",
+          "host": "$CI_ENVIRONMENT_SLUG-login-chart-dashboard-pg.review-apps",
+          "username": "postgres",
+          "password": "postgres"
+        },
+        "newrelic": {
+          "enabled": "false"
+        },
+        "samlSpIssuer": "https://$CI_ENVIRONMENT_SLUG-review-app-dashboard.review-app.identitysandbox.gov",
+        "idpUrl": "https://$CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov",
+        "idpSpUrl": "https://$CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov",
+        "postLogoutUrl": "https://$CI_ENVIRONMENT_SLUG-review-app-dashboard.review-app.identitysandbox.gov",
+        "domainName": "$CI_ENVIRONMENT_SLUG-review-app-dashboard.review-app.identitysandbox.gov"
+      }
       EOF
       )
-    - git clone -b main --single-branch https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.login.gov/lg-public/identity-idp-helm-chart.git
+    - git clone -b lrappaport/env-refactor --single-branch https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.login.gov/lg-public/identity-idp-helm-chart.git
```

Oh make sure to swap this back
```suggestion
    - git clone -b main --single-branch https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.login.gov/lg-public/identity-idp-helm-chart.git
```